D/DecompUtil/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "DecompUtil"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/dev10110/DecompUtil_C.git", "982449a84f620488fe9d954eb9a6d301dddfff3f")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+apk add eigen-dev
+cd DecompUtil_C/
+mkdir build && cd build
+Eigen3_DIR=/usr/share/cmake/Modules cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release .. 
+make install -j
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "macos"; )
+]
```

Comment:
```suggestion
platforms = supported_platforms()
```

Reply:
the buildkite tests still fails on windows, so I've removed windows from the list of supported platforms.

Reply:
Here is the build report for windows: https://buildkite.com/julialang/yggdrasil/builds/1092#0186801a-2ad5-4ddb-aaa6-b06c663bc10c

It seems `M_PI` is not defined in this environment. Do you know how to fix this?

Reply:
https://learn.microsoft.com/en-us/cpp/c-runtime-library/math-constants?view=msvc-170

Reply:
cool thanks for sharing!